integration/util/scenario/ctx.js

```diff
@@ -68,10 +68,13 @@ class Ctx {
     return process.env['LOG'] || this.scenInfo['log_level'];
   }
 
+  __deepColor() {
+    return !!process.env['DEEP_COLOR'];
+  }
+
   __abort(msg) {
-    console.error(msg);
     this.error(msg);
-    process.exit(1);
+    throw(msg);
```

Comment:
Not sure why we changed this (or why we had it). I had issues with the tests not exiting when a node failed, but maybe this is right now?

Reply:
Was it doing `throw` before, or you think `throw` is actually broken? We were logging like 3x so I replaced 1 of them and the process exit with a `throw` since it seemed cleaner